.github/workflows/e2e-fork.yml

```diff
@@ -0,0 +1,45 @@
+name: Tests / E2E Fork
```

Comment:
this is a fork-only workflow that will build and run the test locally to the host. There is no interaction with our ghcr registries.